modules/configuration/ajax/updateProject.php

```diff
@@ -52,6 +54,18 @@
         ),
         array("ProjectID" => $_POST['ProjectID'])
     );
+    $projectID = $_POST['ProjectID'];
+}
+
+$db->delete('project_subproject_rel', array('ProjectID' => $projectID));
+foreach ($_POST['SubprojectIDs'] as $sid) {
+    $db->insertIgnore(
+        'project_subproject_rel',
+        array(
+         'ProjectID'    => $projectID,
+         'SubprojectID' => $sid,
+        )
+    );
```

Comment:
What is the purpose of this?
Are you deleting all association for a project before re-inserting them all?

Reply:
yes instead of comparing before and after the submission

Reply:
Then all `ProjectSubprojectRelID` will change with every update. 
This is not good.

Reply:
good point that rel column was added after this PR was created, I'll have to change the logic

Reply:
@PapillonMcGill fixed